.travis.yml

```diff
@@ -1,36 +1,25 @@
-before_install: 
- - cd PowerUp
- - chmod +x gradlew
 language: android
+sudo: required
 jdk: oraclejdk8
-# Turn off caching to avoid any caching problems
 cache: false
-sudo: true
 
-env:
-  matrix:
-    - ANDROID_API_LEVEL=23 ADB_INSTALL_TIMEOUT=12 ANDROID_BUILD_TOOLS_VERSION=23.0.1 ANDROID_ABI=armeabi-v7a ANDROID_TAG=google_apis
 android:
- components:
-    - tools # to get the new repository-11.xml
+  components:
     - platform-tools
-    - tools # to install Android SDK tools
-    - build-tools-$ANDROID_BUILD_TOOLS_VERSION
-    - android-$ANDROID_API_LEVEL    
-    # For Google APIs
-    - addon-google_apis-google-$ANDROID_API_LEVEL
-    # Google Play Services
+    - tools
+    - build-tools-23.0.1
+    - android-23
     - extra-google-google_play_services
-    # Support library
-    - extra-android-support
-    # Latest artifacts in local repository
     - extra-google-m2repository
     - extra-android-m2repository
-    # Specify at least one system image,
-    - sys-img-armeabi-v7a-android-22
-    - sys-img-armeabi-v7a-android-17
+
+before_script:
+  - cd PowerUp
+  - chmod +x gradlew
+
 script:
- - chmod +x ./gradlew    
```

Comment:
The problem is here.
If you check the Gradle [documentation](https://docs.gradle.org/current/userguide/gradle_wrapper.html) you can see the format of running gradlew command on linux (travis) is `./gradlew <task> `.
No task is being mentioned here.

I haven't tried to fail the test because I know its working fine now. You can check this yourself by comparing the logs of this build and previous builds in travis.

For this PR the app is build see [here](https://travis-ci.org/systers/powerup-android/jobs/285744298#L2091).
For other PRs the app is not being build see [here](https://travis-ci.org/systers/powerup-android/jobs/283878164#L2785)

Reply:
Also chmod command is being used here and at the top as well. I don't think that is needed. (Unrelated thought 😄 )

Reply:
Great !! Sounds good to me  :)